src/SshNetTests/SftpTests.cs

```diff
@@ -4242,12 +4242,14 @@ public void Sftp_SftpFileStream_Seek_BeyondEndOfFile()
                         fs.WriteByte(0x04);
                     }
 
-
                     // seek beyond EOF but not beyond buffer size
                     // do not write anything
                     using (var fs = client.OpenWrite(remoteFile))
                     {
-                        fs.Seek(3L, SeekOrigin.Begin);
+                        var newPosition = fs.Seek(3L, SeekOrigin.Begin);
+
+                        Assert.AreEqual(3, newPosition);
+                        Assert.AreEqual(newPosition, fs.Position);
```

Comment:
Possibly explicit expected value will be cleared.

```suggestion
                        Assert.AreEqual(3, fs.Position);
```

Reply:
Fixed in upcoming commit.